subscribe/lambda/function.rb

```diff
@@ -33,6 +33,21 @@ def subscribe_all!(lambda = Aws::Lambda::Client.new, cloudwatch = Aws::CloudWatc
             end
           end
         end
+
+        private
+
+        def all_functions(lambda)
+          marker = nil
+          functions = []
+
+          loop do
+            response = lambda.list_functions(marker: marker)
+            functions << response.functions
+            break unless (marker = response.next_marker)
```

Comment:
This may not work at all, but what do you think about using an `until` loop for readability?

```ruby
response = lambda.list_functions
next_marker = response.next_marker
functions = response.functions

until next_marker.nil?
  page = lambda.list_functions(marker: next_marker)
  next_marker = page.next_marker
  functions.append(*page.functions)
end
```

Reply:
Nice find, I'll try that out.

Reply:
It works great and greatly simplifies this method. Finally a use for `flat_map` 🤤